unit-tests/src/test/java/org/eclipse/collections/impl/list/mutable/FastListTest.java

```diff
@@ -1137,7 +1137,7 @@ public void testLazyCollectForEach()
     @Test
     public void testLazyFlattenForEach()
     {
-        FastList<Integer> list = (FastList<Integer>) Interval.oneTo(5).toList();
+        MutableList<Integer> list = Interval.oneTo(5).toList();
```

Comment:
Same comment on explicit construction of `FastList`, since this is in `FastListTest`.